spec/build.xml

```diff
@@ -230,7 +230,7 @@
     </target>
 
     <target name="generate" description="Generate the specification"
-	    depends="generate-schema-archives,validate-tt-specs,html">
+	    depends="generate-schema-archives,validate-tt-specs,html,kwexpand">
```

Comment:
don't you have to do `kwexpand` before `html`?

Reply:
Dependencies are executed in the order they appear in the list

Reply:
That's what I thought, which is why I expected `kwexpand` to be before `html`. That way the expanded entities are available to the script that generates the html from the xml. I don't think `kwexpand` touches the generated html.

Reply:
Kwexpand is rewritten and modifies ttml1.html

Reply:
Right, I understand now, sorry I failed to spot that.